nodejs/Auto_Increment_Version.yml

```
@@ -0,0 +1,56 @@
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          path: pr-branch
+
+      - name: Compare package.json versions
+        id: compare-versions
+        run: |
+          MAIN_VERSION=$(jq -r .version main-branch/package.json)
+          PR_VERSION=$(jq -r .version pr-branch/package.json)
+          echo "Main branch version: $MAIN_VERSION"
+          echo "PR branch version: $PR_VERSION"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)